ios/CodePush/CodePush.m

```diff
@@ -760,4 +760,29 @@ - (void)applicationWillResignActive
     resolve(nil);
 }
 
+#pragma mark - Methods for handling dispatching of download progress events to JS (Private)
```

Comment:
The reason why I didn't do that is because `sendDownloadProgressDuringNextFrame` is not part of the `RCTFrameUpdateObserver` protocol, its a custom method